pkg/approver/manager/subjectaccessreview.go

```diff
@@ -50,24 +54,107 @@ func NewSubjectAccessReview(client client.Client, evaluators []approver.Evaluato
 // Review will evaluate whether the incoming CertificateRequest should be
 // approved. All evaluators will be called with CertificateRequestPolicys that
 // have been RBAC bound to the user included in the CertificateRequest.
-func (s *subjectaccessreview) Review(ctx context.Context, cr *cmapi.CertificateRequest) (bool, string, error) {
+func (s *subjectaccessreview) Review(ctx context.Context, cr *cmapi.CertificateRequest) (ReviewResponse, error) {
 	crps := new(cmpapi.CertificateRequestPolicyList)
 	if err := s.client.List(ctx, crps); err != nil {
-		return false, "", err
+		return ReviewResponse{}, err
+	}
+
+	// If no CertificateRequestPolicies exist in the cluster, return
+	// ResultUnprocessed.
+	if len(crps.Items) == 0 {
+		return ReviewResponse{Result: ResultUnprocessed, Message: "No CertificateRequestPolicies exist"}, nil
+	}
+
+	boundPolicies, err := s.boundPolicies(ctx, cr, crps.Items)
+	if err != nil {
+		return ReviewResponse{}, fmt.Errorf("failed to determine bound policies: %w", err)
+	}
+
+	// If no policies are bound to the requesting user, return denied.
```

Comment:
I agree. In the next PR this functionality is changed so that it returns Unprocessed https://github.com/cert-manager/policy-approver/pull/19/files#diff-d9f31446044d6bf22cf575a8f70e3c0f4e66a5992ad155845cd96eb827ddfc7bR77